src/scripts/pokemons/PokemonList.ts

```diff
@@ -1911,11 +1927,11 @@ const pokemonList = createPokemonArray(
     {
         'id': 45.02,
         'name': 'Pinkan Vileplume',
-        'catchRate': 45,
+        'catchRate': 20,
```

Comment:
Was this intentional?

Reply:
No. I have undone all 3 of those mistakes